### ISSUE TITLE
note: ld: error: unable to find library -lgcc

### ISSUE THREAD

**Issue by xiaoheiai4719** [brickpop/flutter-rust-ffi#25]
when i  use make all 
  = note: ld: error: unable to find library -lgcc
          clang-12: error: linker command failed with exit code 1 (use -v to see invocation)

error: aborting due to previous error

error: could not compile `example-rs`

To learn more, run the command again with --verbose.
make: *** [target/aarch64-linux-android/release/libexample.so] Error 101

**Comment by brickpop** [comment]
What platform are you running this on?
Did you install and point to the right Android SDK/NDK build tools?

**Comment by xiaoheiai4719** [comment]
Mac OS， i install the NDK from android Studio。and 
![image](https://user-images.githubusercontent.com/5488138/123790383-d9b2a600-d910-11eb-8ed9-f50f46363478.png)
![image](https://user-images.githubusercontent.com/5488138/123790441-eafbb280-d910-11eb-9168-aa5d89ab8d68.png)

**Comment by xiaoheiai4719** [comment]
version  22.1.7171670  is ok

**Comment by alexandru-calinoiu** [comment]
Yes, looks like the version NDK version 23 will fail, try @xiaoheiai4719 suggestion.

**Comment by VioPaige** [comment]
Is there any better solution for this yet than to simply downgrade? I'm using a framework that requires me to use NDK 25.0.8775105, but I then run into a whole bunch of errors:
```
= note: rust-lld: error: unable to find library -llog
          rust-lld: error: unable to find library -landroid
          rust-lld: error: unable to find library -ldl
          rust-lld: error: unable to find library -llog
          rust-lld: error: unable to find library -lgcc
          rust-lld: error: unable to find library -ldl
          rust-lld: error: unable to find library -lc
          rust-lld: error: unable to find library -lm
```
